pkg/codegen/schema/schema.go

```diff
@@ -1307,15 +1410,28 @@ func (t *types) bindEnumType(token string, spec ComplexTypeSpec) (*EnumType, err
 }
 
 func bindTypes(pkg *Package, complexTypes map[string]ComplexTypeSpec) (*types, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	ctx, err := plugin.NewContext(nil, nil, nil, nil, cwd, nil, false, nil)
+	if err != nil {
+		return nil, err
+	}
+	// TODO: do we need to manually close the context?
```

Comment:
Yes, we should close this context. Though it looks like it's not used?

Reply:
It's used to instantiate the loader. I was using the commented out `defer` statement initially, but it causes a panic if more than one plugin is loaded.

Reply:
Don't we only instantiate a loader if none is passed in?

Reply:
Talked to @pgavlin offline about this -- I'm going to refactor to instantiate the loader at the top level, and then pass that loader instance down to the binding functions.